packages/generator/README.md

```diff
@@ -146,6 +146,16 @@ const config = {
 
 When the specified amount of memory is reached, Redis can be configured to apply an eviction policy. Refer to [this article](https://redis.io/topics/lru-cache/) to setup Redis as an LRU cache and to learn more about supported eviction policies. 
 
+## Logging
+Default log level of the SDK is WARN (warning). SDK uses [loglevel](https://www.npmjs.com/package/loglevel) npm package. All the log levels supported by [loglevel](https://www.npmjs.com/package/loglevel) package are supported in SDK.
+
+To change the loglevel, set the desired level on the SDK logger.
+```javascript
+import * as CommerceSdk from "commerce-sdk";
+import * as log from "loglevel";
+
+log.getLogger(CommerceSdk.COMMERCE_SDK_LOGGER_KEY).setLevel(log.levels.INFO);
```

Comment:
Hmm, I don't like having to take two components to do something like this.  I would suggest only needing 'commerce-sdk' or only needing 'loglevel' to modify the level.  Otherwise, it isn't crystal clear to the end-user.  

I don't really care which way you go but I suggest we have it be something like
```
import { Logger } from "commerce-sdk"
Logger.setLevel(Logger.INFO);
```
OR something
```
import * as log from "loglevel"
log.getLogger("COMMERCE-SDK").setLevel(log.levels.INFO);
```

What we should avoid is showing you need two distinct components to set something like the logging level.

(Note: I somewhat prefer the first option, but could be convinced either way)

Reply:
I too prefer the first one. It matches with the way that we use to use the logger